aim/web/ui/src/components/CustomTable/Table.tsx

```diff
@@ -220,80 +220,93 @@ function Table(props) {
             'Table--grouped': groups,
           })}
         >
+          {!groups && props.multiSelect && Array.isArray(props.data) && (
+            <ErrorBoundary key={'selection'}>
+              <div
+                className={classNames({
+                  Table__pane: true,
+                  'Table__pane--selection': true,
+                  onlyGroupColumn: leftPane.length === 0,
+                })}
+              >
+                <Column
+                  topHeader={true}
+                  showTopHeaderContent={true}
+                  showTopHeaderBorder={true}
+                  col={{
+                    isHidden: false,
+                    key: 'selection',
+                    pin: 'left',
+                    topHeader: '',
+                    content: (
+                      <Checkbox
+                        color='primary'
+                        size='small'
+                        className={classNames('Table__column__selectCheckbox', {
+                          Table__column__headerCheckbox:
+                            props.data[0]?.rowMeta?.color,
```

Comment:
Please assign a value from props.data[0]?.rowMeta?.color some variable, to access them faster and make code more readable. You have used this excretion three times this is a good signal for assigning expression to a variable.